src/APIActionCreator.es6.js

```diff
@@ -23,6 +23,9 @@ const ActionCreator = require('./ActionCreator.es6');
 let defaultHeaders = {
 	headers: undefined // no headers by default
 };
+let defaultBaseURL = {
```

Comment:
why is this an object?

Oh, I guess bc default headers? Idk why that is either. Would you mind making a `defaultOptions` object w/ headers & url? Should be cleaner and less stuttering when reading.